src/GenericHostBuilderExample/GenericHostBuilderExample.csproj

```diff
@@ -14,8 +14,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.3.1" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.0.0-preview8.19405.4" />
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.4.0" />
```

Comment:
Will this work as-is or does it need to be a beta package at the moment?

Reply:
It would work as it is I just wanted to show-case it with the latest version. Should I downgrade it again?

Reply:
No, it's fine to use the latest version. I just wasn't sure if this required any of the latest changes that you've been working on in the PR.